internal/internal_worker.go

```diff
@@ -1383,11 +1383,12 @@ func isError(inType reflect.Type) bool {
 	return inType != nil && inType.Implements(errorElem)
 }
 
-func getFunctionName(i interface{}) string {
+func getFunctionName(i interface{}) (name string, isMethod bool) {
 	if fullName, ok := i.(string); ok {
-		return fullName
+		return fullName, false
 	}
 	fullName := runtime.FuncForPC(reflect.ValueOf(i).Pointer()).Name()
+	isMethod = strings.ContainsAny(fullName, "*")
```

Comment:
Would be nice to add a comment that explains what "*" means.

Reply:
Oh man, this is how you did it? Can we guarantee that `*` can never appear in a function name except as part of a receiver pointer? This feels pretty icky, but if it's the only reason then I suppose it's reasonable.

Definitely needs the comment for sure, otherwise could be deeply confusing to future readers